app/internal/search.py

```diff
@@ -16,7 +16,7 @@ def get_stripped_keywords(keywords: str) -> str:
 
 
 def get_results_by_keywords(
-                            session: SessionLocal,
+                            session: Session,
```

Comment:
Please use Depends(get_db) (also in other files, if applicable)

Reply:
I don't think i fully understand your request here.
As I've mentioned before, you can see in [This comment by Tiangolo](https://github.com/tiangolo/fastapi/issues/1105#issuecomment-609919850), that Depends is not meant for use outside of path operation. Additionally, this function doesn't call a session directly, but is rather passed one from a path operation (where it is created using Depends(get_db)).
I should also mentioned that this is not part of my code and I merely fixed the incorrect type hinting in this module.

Reply:
I see, you are right. Sababa, let's keep it this way :)